src/components/MenuBars/MenuBars.tsx

```diff
@@ -170,64 +210,81 @@ export const MenuBars = ({showPreviousColumn, showNextColumn, onPreviousColumn,
         >
           {fabIsExpanded ? <CloseIcon aria-hidden /> : <MenuIcon aria-hidden />}
         </button>
+
+        {/* role=any: toggle ready, toggle raise hand, options */}
         {(fabIsExpanded || isReady || raisedHand) && (
           <ul
-            className={classNames("menu-bars-mobile__options menu-bars-mobile__options--vertical", {
+            className={classNames("menu-bars-mobile__options menu-bars-mobile__options--horizontal", {
               "menu-bars-mobile__options--isExpanded": fabIsExpanded,
               "menu-bars-mobile__options--hasActiveButton": isReady || raisedHand,
             })}
           >
             {(fabIsExpanded || isReady) && (
-              <li className="menu-bars-mobile__fab-option menu-bars-mobile__fab-option--vertical">
+              <li className={classNames("menu-bars-mobile__fab-option", "menu-bars-mobile__fab-option--horizontal", {"menu-bars-mobile__fab-option--active": isReady})}>
                 <TooltipButton
                   active={isReady}
-                  direction="right"
+                  direction="left"
                   label={isReady ? t("MenuBars.unmarkAsDone") : t("MenuBars.markAsDone")}
                   icon={CheckIcon}
                   onClick={toggleReadyState}
                 />
               </li>
             )}
             {(fabIsExpanded || raisedHand) && (
-              <li className="menu-bars-mobile__fab-option menu-bars-mobile__fab-option--vertical">
+              <li className={classNames("menu-bars-mobile__fab-option", "menu-bars-mobile__fab-option--horizontal", {"menu-bars-mobile__fab-option--active": raisedHand})}>
                 <TooltipButton
                   active={raisedHand}
-                  direction="right"
+                  direction="left"
                   label={raisedHand ? t("MenuBars.lowerHand") : t("MenuBars.raiseHand")}
                   icon={RaiseHand}
                   onClick={toggleRaiseHand}
                 />
               </li>
             )}
             {fabIsExpanded && (
-              <li className="menu-bars-mobile__fab-option menu-bars-mobile__fab-option--vertical">
-                <TooltipButton direction="right" label={t("MenuBars.settings")} onClick={showSettings} icon={SettingsIcon} />
+              <li
+                className={classNames("menu-bars-mobile__fab-option", "menu-bars-mobile__fab-option--horizontal", {"menu-bars-mobile__fab-option--active": showBoardReactionsMenu})}
+              >
+                <TooltipButton
+                  active={showBoardReactionsMenu}
+                  direction="left"
+                  label={t("MenuBars.openBoardReactionMenu")}
+                  icon={BoardReactionIcon}
+                  onClick={toggleBoardReactionsMenu}
```

Comment:
for some reason, clicking this button again to close the reaction menu doesn't work for me on mobile

Reply:
fixed